web/src/components/njwds-layout/PageSkeleton.test.tsx

```diff
@@ -9,27 +8,27 @@ describe("<PageSkeleton />", () => {
     jest.resetAllMocks();
   });
 
-  it("does not render report an issues bar if onboarding is not completed", () => {
-    useMockBusiness({ onboardingFormProgress: "UNSTARTED" });
-
+  it("does not render the beta bar, legal message, and skip main content", () => {
```

Comment:
[dust] Maybe specify the location is specific to the landing page in the test name